### PR TITLE
filebeat support in logging role

### PR DESCRIPTION
**Pull request by zfeldstein**
With LSF being deprecated this role adds support to install and configure filebeat.

### CODE REVIEW

File: roles/logging-config/handlers/main.yml [hunk 1]
```diff
@@ -2,3 +2,6 @@
 - name: restart logstash-forwarder
   service: name=logstash-forwarder state=restarted must_exist=false
   failed_when: false
+
+- name: restart filebeat
+  service: name=filebeat state=restarted must_exist=false
```
Comment (paulczar) [inline roles/logging-config/handlers/main.yml]: currently must_exist doesn't function with systemd units, may need a failed_when: false in here.

File: roles/logging/defaults/main.yml [hunk 2]
```diff
@@ -13,12 +13,16 @@ logging:
       cluster_name: "unknown"
     logs:
       - paths:
-          - /var/log/syslog
+          - "{{ ( os == 'ubuntu' )| ternary('/var/log/syslog', '/var/log/messages') }}"
```
Comment (paulczar) [inline roles/logging/defaults/main.yml]: :+1: such ternary

**Comment by sandfish8** [comment]
👍

**Comment by paulczar** [comment]
retest

**Comment by zfeldstein** [comment]
retest

**Comment by zfeldstein** [comment]
retest

**Comment by zfeldstein** [comment]
retest

**Comment by zfeldstein** [comment]
retest

**Comment by zfeldstein** [comment]
retest

**Comment by zfeldstein** [comment]
retest

**Comment by zfeldstein** [comment]
retest

**Comment by zfeldstein** [comment]
retest

**Comment by zfeldstein** [comment]
retest

**Comment by scoop206** [comment]
retest

**Pull request merged**